deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -7,16 +7,135 @@ metadata:
     operatorframework.io/cluster-monitoring: "true"
     operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
     repository: https://github.com/openshift/windows-machine-config-operator
-  name: windows-machine-config-operator.v0.0.0
+  name: windows-machine-config-operator.v1.0.0-rc1
   namespace: openshift-windows-machine-config-operator
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions: {}
-  description: Placeholder description
+  description: |-
+    Issues with this distribution of WMCO can be opened against the [WMCO repository](https://github.com/openshift/windows-machine-config-operator).
+    Please read through the [troubleshooting doc](https://github.com/openshift/windows-machine-config-operator/blob/community-4.6/docs/TROUBLESHOOTING.md)
+    before opening an issue
+
+    Please ensure that when installing this operator the startingCSV you subscribe to is supported on the
+    version of OKD/OCP you are using. This CSV is meant for OKD/OCP 4.6.
+
+    Upgrading the OKD/OCP cluster from version 4.6 to 4.7 while running this operator is not supported. When upgrading
+    a cluster, remove all Windows MachineSets and uninstall this operator. Once the cluster upgrade process has completed,
+    you may reinstall the operator, and recreate all Windows MachineSets.
+
+    ## Documentation
+
+    ### Introduction
+    The Windows Machine Config Operator configures Windows Machines into nodes, enabling Windows container workloads to
+    be ran within OKD/OCP clusters. The operator is configured to watch for Machines with a `machine.openshift.io/os-id: Windows`
+    label. The way a user will initiate the process is by creating a MachineSet which uses a Windows image with the Docker
+    container runtime installed. The operator will do all the necessary steps to configure the underlying VM so that it can join
```

Comment:
```suggestion
    container runtime installed. The operator completes all the necessary steps to configure the underlying VM so that it can join
```